Stripe/StripeError+Private.h

```diff
@@ -0,0 +1,15 @@
+//
+//  StripeError+Private.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/22/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSError (STPCustomerContext)
```

Comment:
Having `+Private.h` but the category actually being `(STPCustomerContext)` is confusing. Should just make a catch-all private (and maybe move any other related internal only error methods here) or rename the file.